clusterloader2/testing/density/config.yaml

```diff
@@ -37,16 +41,16 @@ steps:
     - basename: saturation-rc
       objectTemplatePath: rc.yaml
       templateFillMap:
-        Replicas: {{$replicas}}
+        Replicas: {{$PODS_PER_NAMESPACE}}
         PodKind: saturationPod
 - measurements:
   - Identifier: WaitForSaturationRunning
     Method: WaitForRunningPods
     Params:
-      desiredPodCount: {{$totalPodsRounded}}
+      desiredPodCount: {{$totalPods}}
```

Comment:
I'm not convinced we should do this - I think that more standard our tests are as a whole, the better.

As a result, I would prefer using the same measurements in both tests when possible.
So I think we should use:
WaitForControlledPodsRunning (given that we have RCs here anyway).

And then you won't need totalPods'
The same observation is for latencyPods.

Reply:
Ok. I see your point.
I assume that this PR is blocked until #264 is merged. Or should I use the old version of WaitForControlledPodsRunning?

Reply:
Yes - it's blocked.